README.md

```diff
@@ -43,6 +33,83 @@ Browser support
 
 `memdown` requires a ES5-capable browser. If you're using one that's isn't (e.g. PhantomJS, Android < 4.4, IE < 10) then you will need [es5-shim](https://github.com/es-shims/es5-shim).
 
+## Data types
+
+Unlike [`leveldown`], `memdown` does not stringify keys or values. This means that in addition to Buffers, you can store any JS type without the need for [`encoding-down`]. The only exceptions are `null` and `undefined` values. These are converted to empty strings.
+
+```js
+const db = levelup(memdown())
+
+db.put(12, true, (err) => {
+  if (err) throw err
+
+  db.createReadStream({
+    keyAsBuffer: false,
+    valueAsBuffer: false
+  }).on('data', (entry) => {
+    console.log(typeof entry.key) // 'number'
+    console.log(typeof entry.value) // 'boolean'
+  })
+})
+```
+
+Practically speaking, few types are suitable for keys. You can use Buffers or strings, which sort lexicographically, or numbers or Dates, which sort naturally. That's about it.
```

Comment:
this should be up to the consumer to decide, can we reword this as a hint?

Reply:
I'm all for letting the consumer decide, but what other types are there? That sort in a sensible way without needing a custom comparator (which btw, might be a nice feature to have?).

Reply:
maybe they don't need it to sort at all

Reply:
If it doesn't sort (specifically: if two keys are equal) then `get()` doesn't work, because `functional-red-black-tree` won't know which node to return.

Reply:
Actually no, I'm wrong there, as `put` will update the node.

Reply:
Suggestions for new wording?

Reply:
I think we don't need to mention this at all, people won't think about custom key types unless they have something special in mind anyway, in which case they probably know enough about the platform to know the limitations.

Reply:
That's true. On the other hand, if we inform people (newcomers) of the possibilities, we might inspire them :) So I prefer your earlier suggestion of rewording as a hint.

How about this? Move it to the paragraph above it, in a more "optimistic" tone, and also segway into a note about `null` and `undefined` keys (which I now realize is missing):

---

Unlike [`leveldown`], `memdown` does not stringify keys or values. This means that in addition to Buffers, you can store any JS type without the need for [`encoding-down`]. For keys for example, you could use Buffers or strings, which sort lexicographically, or numbers, even Dates, which sort naturally. The only exceptions are `null` and `undefined`. Keys of that type are rejected; values of that type are converted to empty strings.

[`encoding-down`]: https://github.com/Level/encoding-down
[`leveldown`]: https://github.com/Level/leveldown

Reply:
Yes, this ^